ross/fluid_flow/fluid_flow.py

```diff
@@ -0,0 +1,400 @@
+    ...                                          radius_stator, xi, yi,  visc, rho, plot_eccentricity=True)
+    >>> P = my_pressure_matrix.calculate_pressure_matrix()
+    >>> my_pressure_matrix.plot_pressure_z(show_immediately=False)
+    >>> my_pressure_matrix.plot_shape(show_immediately=False)
+    >>> my_pressure_matrix.plot_pressure_theta_cylindrical(z=int(nz/2), show_immediately=False)
+    >>> my_pressure_matrix.plot_pressure_theta(z=int(nz/2), show_immediately=False)
+    >>> plt.show()
+    >>> plt.close('all')
+
+    """
+    def __init__(self, nz, ntheta, nradius, length, omega, p_in,
+                 p_out, radius_rotor, radius_stator, xi,
+                 yi, visc, rho, plot_eccentricity=False):
+        self.nz = nz
+        self.ntheta = ntheta
+        self.nradius = nradius
+        self.n_interv_z = nz - 1
+        self.n_interv_theta = ntheta - 1
+        self.n_interv_radius = nradius - 1
+        self.length = length
+        self.ltheta = 2.*np.pi
+        self.dz = length / self.n_interv_z
+        self.dtheta = self.ltheta / self.n_interv_theta
+        self.ntotal = self.nz * self.ntheta
+        self.omega = omega
+        self.p_in = p_in
+        self.p_out = p_out
+        self.radius_rotor = radius_rotor
+        self.radius_stator = radius_stator
+        self.xi = xi
+        self.yi = yi
+        self.visc = visc
+        self.rho = rho
+        self.re = np.zeros([self.nz, self.ntheta])
+        self.ri = np.zeros([self.nz, self.ntheta])
+        self.z = np.zeros([1, self.nz])
+        self.xre = np.zeros([self.nz, self.ntheta])
+        self.xri = np.zeros([self.nz, self.ntheta])
+        self.yre = np.zeros([self.nz, self.ntheta])
+        self.yri = np.zeros([self.nz, self.ntheta])
+        self.p_mat = np.zeros([self.nz, self.ntheta])
+        self.bearing_seal_type = -1
+        self.plot_counter = 0
+        self.calculate_coefficients(plot_eccentricity)
+        self.pressure_matrix_available = False
+        self.distance_between_centers = np.sqrt((xi**2 + yi**2))
+        self.difference_between_radius = radius_stator - radius_rotor
+        self.eccentricity_ratio = self.distance_between_centers/self.difference_between_radius
+
+    def calculate_pressure_matrix(self):
+        """This function calculates the pressure matrix
+        """
+        if self.bearing_seal_type == 0:
+            for i in range(self.nz):
+                for j in range(self.ntheta):
+                    self.p_mat[i][j] = ((-3*self.visc*self.omega)/self.difference_between_radius**2)*\
+                                       ((i*self.dz)**2 - (self.length**2)/4)*\
+                                       (self.eccentricity_ratio*np.sin(j*self.dtheta))/\
+                                       (1 + self.eccentricity_ratio*np.sin(j*self.dtheta))**3
+            self.pressure_matrix_available = True
+        return self.p_mat
+
+    def calculate_coefficients(self, plot_cut=False):
```

Comment:
This function is doing some calculation and making a plot. I think it would be better to separate that and make a different function for the plot functionality.
Additionally, when using a function to create a plot with matplotlib, it is better to make the axes an argument:
```python
def foo(ax=None):
    # if axes is not given get the one from the current figure
    if ax is None:
        ax = plt.gca()
    .
    .
    ax.plot(...)
    .
    .
    return ax
```
For more details about matplotlib objects (figure, axes, axis etc.) refer to https://matplotlib.org/tutorials/introductory/usage.html

Reply:
Ok. I'm working on it.

Reply:
I'm working on creating a new plot function to separate it from this function.
As for these changes in ploting functions, we will soon change from matplotlib to bokeh, so I think we can let them like this for now. What do you think?